google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudType.java

```diff
@@ -118,8 +118,8 @@ public void connect(@NotNull ConnectionCallback<AppEngineDeploymentConfiguration
 
       if (!Services.getLoginService().isLoggedIn()) {
         callback.errorOccurred(GctBundle.message("appengine.deployment.error.not.logged.in"));
-      } else if (CloudSdkUtil.isCloudSdkExecutable(
-          CloudSdkUtil.toExecutablePath(CloudSdkService.getInstance().getCloudSdkHomePath()))) {
+      } else if (CloudSdkUtil.isCloudSdkExecutable(CloudSdkUtil.toExecutablePath(
+          CloudSdkService.getInstance().getCloudSdkHomePath().toString()))) {
```

Comment:
@patflynn previously agreed that we don't want to prevent the user from creating an invalid SDK. There is an exception: the Cloud UI forces the user to have a valid path. So bottom line is that we can't always assume its set or valid.